summerfi-api/get-morpho-claims-function/src/index.ts

```diff
@@ -45,6 +48,8 @@ export const handler = async (
 
   const claims = await getClaims({
```

Comment:
similar here getClaims has no error handling, basically it will return unhandled 500 status, but it could be handled nicely as we know what can go wrong and provide the message to the client